bin/setup-policy-routes.sh

```diff
@@ -193,78 +193,78 @@ EOF
     _install_and_reload "$work" "$file"
 }
 
+create_if_overrides() {
+    local iface="$1"
+    local tableid="$2"
+    local ether="$3"
+    local cfgfile="$4"
+    local cfgdir="${cfgfile}.d"
+    local dropin="${cfgdir}/eni.conf"
+    mkdir -p "$cfgdir"
+    if [ $tableid -eq 0 ]; then
+        # primary, just match on MAC
+        cat <<EOF > "${dropin}.tmp"
+[Match]
+MACAddress=${ether}
+EOF
+    else
+        # secondary. match on MAC and set up private route tables
+        cat <<EOF > "${dropin}.tmp"
+[Match]
+MACAddress=${ether}
+[DHCPv4]
+RouteTable=${tableid}
+[Route]
+Table=${tableid}
+
+[IPv6AcceptRA]
+RouteTable=${tableid}
+EOF
+    fi
+    mv "${dropin}.tmp" "$dropin"
+    echo 1
+}
+
+add_altname() {
+    local iface=$1
+    local ether=$2
+    local eni_id
+    eni_id=$(get_iface_imds "$ether" interface-id)
+    if [ -n "$eni_id" ] &&
+           ! ip link show dev "$iface" | grep -q -E "altname\s+${ether}"; then
+        ip link property add dev "$iface" altname "$eni_id"
```

Comment:
Is there a good way to make this idempotent? It's not clear to me here if this would survive two calls with the same parameters.